### ISSUE TITLE
Can't use Windows standard key sequence on main menu

### ISSUE THREAD

**Issue by breyed** [Microsoft/calculator#377]
The main menu doesn't allow the standard windows sequence of Alt+letter followed by another letter. As an example of what you should be able to do, you can exit Notepad this way: press Alt+F, then release both keys, and press X. This is written Alt+F, X.

In calculator, if you press Alt+H, 2, it doesn't work. The 2 get entered into the calculator window. If you keep the Alt key held down, spanning the press of the H and the 2, then it works, but this requires an awkward finger stretch.

Calculator's menus should support standard Windows key sequences.

**Device and Application Information:**
 - OS Build: 10.0.17763.0
 - Architecture: X64
 - Application Version: 10.1812.10048.0

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I've seen this issue come in and have gone to tell a human about it.

**Comment by grochocki** [comment]
Thanks for catching this one too! I _beleive_ that addressing #378 should address this as well, so closing this as a dupe for now as we investigate #378 further.
